src/main/java/com/umbr3114/Main.java

```diff
@@ -47,10 +47,14 @@ public static void main(String[] args) {
         /*
          * Auth endpoints
          */
+
         post("/register", AuthRoutes.registerUser, new JsonResponse());
         post("/login", AuthRoutes.loginUser, new JsonResponse());
         get("/logout", AuthRoutes.logoutUser, new JsonResponse());
 
+
+        put("/drops/:dbname/update", DropController.updateDrop);
+        delete("/drops/:dropName/delete", DropController.deteleDrop);
```

Comment:
the :dropName in this path is never used in the route, so it really doesn't have to be here.

it could be `/drops/delete`